docs/reference/api/platform/Error.md

```diff
@@ -196,6 +198,75 @@ mbed_error_status_t configure(int config_value) {
 }
 ```
 
+#### Using `mbed_get_first_error()` and `mbed_get_first_error_info()` functions to retrieve the first error or first warning logged in the system
+
+The code below uses `mbed_get_first_error()` and `mbed_get_first_error_info()` functions to retrieve the first error or first warning logged in the system
+using `MBED_WARNING()/MBED_ERROR()` calls.
+
+```C
+void get_first_error_info() {
+    mbed_error_status_t first_error_status = mbed_get_first_error();
+    printf("\nFirst error code = %d", MBED_GET_ERROR_CODE(first_error_status))
+
+    //Now retrieve more information associated with this error
+    mbed_error_ctx first_error_ctx;
+    mbed_error_status_t first_error = mbed_get_first_error(&first_error_ctx);
+}
+```
+
+#### Using `mbed_get_last_error()` and `mbed_get_first_last_info()` functions to retrieve the last error or last warning logged in the system
```

Comment:
Should `mbed_get_first_last_info()` be `mbed_get_last_info()` ?

Reply:
Good catch, that was a typo, let me fix it.